docs/customization/menu.rst

```diff
@@ -41,32 +40,84 @@ In the example below we are adding a one new item to the Admin panel menu and a
         {
             $menu = $event->getMenu();
 
-            $menu->addChild('backend_main')
-                ->setLabel('Test Backend Main');
+            $newSubmenu = $menu->addChild('new')
+                ->setLabel('New Admin Panel Menu');
```

Comment:
Would be a cool functionality :D But @gabiudrescu may it stay like that or should I change labels? :)